packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -262,18 +261,29 @@ export const renderHtml = async (
             get(_target, name: string) {
               if (isDev) {
                 // TODO too long, we need to refactor this logic
-                let file = filePath.slice(config.basePath.length);
-                file = file.split('?')[0]!;
-                file = file.startsWith('@fs/')
-                  ? file.slice('@fs'.length)
-                  : encodeFilePathToAbsolute(joinPath(opts.rootDir, file));
+                const file = (
+                  filePath.startsWith('/@fs/')
```

Comment:
```suggestion
                  filePath.slice(config.basePath.length).startsWith('@fs/')
```

(A note: we try to make it work with `basePath`, but as we don't have tests, there might be some bugs as of now.)